.github/workflows/test.yml

```diff
@@ -45,9 +47,24 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ matrix.rust }}
+          target: ${{ matrix.target }}
           override: true
+      - name: Install cargo cross
+        run: |
+          curl -L https://github.com/cross-rs/cross/releases/download/v0.2.4/cross-x86_64-unknown-linux-gnu.tar.gz | tar xvz
+          sudo mv cross cross-util /usr/bin
+          which cross
```

Comment:
In https://github.com/actions-rs/cargo#cross-compilation that you linked it says that it will install `cross` automatically. Did that not work here?

Reply:
it does work but uses `cargo install cross` I think which just takes longer :man_shrugging: will change it though